<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Memory size:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memorySizeLE">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>30</y>
     <width>83</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="algorithmsGBX">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorithms</string>
   </property>
   <widget class="QRadioButton" name="FFRB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>First Fit</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="BFRB">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Best Fit</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>61</y>
     <width>561</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Holes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTableWidget" name="holeTable">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Hole address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hole size</string>
          </property>
         </column>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="addHoleBTN">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add hole</string>
         </property>
        </widget>
        <widget class="QPushButton" name="deleteHoleBTN">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete hole</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearHoleBTN">
         <property name="text">
          <string>Clear Holes</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startBTN">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start simulation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Processes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTableWidget" name="processTable">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Process size</string>
          </property>
         </column>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="addProcessBTN">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add process</string>
         </property>
        </widget>
        <widget class="QPushButton" name="deleteProcessBTN">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete process</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearProcessBTN">
         <property name="text">
          <string>Clear Processes</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitBTN">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
